.github/labeler.yml

```diff
@@ -0,0 +1,22 @@
+# This is used with the label workflow which
+# will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# For more information, see:
+# https://github.com/actions/labeler
+
+'package: core':
+  - core/**/*
+
+'package: angular':
+  - angular/**/*
+  - packages/angular-server/**/*
+
+'package: react':
+  - packages/react/**/*
+  - packages/react-router/**/*
+
+'package: vue':
+  - vue/**/*
+  - packages/vue/**/*
+  - packages/vue-router/**/*
```

Comment:
Minor thing, but are we able to do like `packages/vue-*`? Just in case we have `@ionic/vue-server` for SSR we won't need to update this

Reply:
I tested it out and it does seem to work, so I can make that change. Should we do the same for react/angular? Or leave it?

Reply:
Should I remove the first check for vue? Are we deleting all of the old stuff?

Reply:
Yeah you can remove the first check -- it will be removed when Ionic Vue launches

Reply:
might as well do it for the other one in case we add something